velox/docs/functions/spark/json.rst

```diff
@@ -22,6 +22,8 @@ JSON Functions
 
 .. spark:function:: get_json_object(json, path) -> varchar
 
-    Extracts a json object from path::
+    Extracts a json object from ``path``. Returns NULL if it finds json string
+    is malformed. ::
 
-        SELECT get_json_object('{"a":"b"}', '$.a'); -- b
\ No newline at end of file
+        SELECT get_json_object('{"a":"b"}', '$.a'); -- 'b'
+        SELECT get_json_object('{"a":{"b":"c"}}', '$.a'); -- '{"b":"c"}'
```

Comment:
Does this function take an argument of type VARCHAR and return result of type JSON? (this is what signature on L23 suggests)? Or, does this function return VARCHAR result?

Reply:
All inputs are VARCHAR and output is also VARCHAR. Just clarified in the doc. Thanks!